### PR TITLE
TDP-3838 Add the error message when voice-v2 rate-limit is reached

### PR DESCRIPTION
**Pull request by muesono-daitan**
## For the Committer

⚠️ Ensure that for this repo (**bandwidth.github.io**) that the pull request change is opened against the branch `stop-gap-v2`

### CODE REVIEW

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-3838.s3-website-us-east-1.amazonaws.com/

**Review by ghost — APPROVED** [review]
That looks good.  Unrelated to this change, it would be good to refer to this from the <Transfer> page as an example of why it's very important to provide a transferCompleteUrl

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-3838.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-3838.s3-website-us-east-1.amazonaws.com/

File: voice/bxml/verbs/transfer.md [hunk 1]
```diff
@@ -16,8 +16,9 @@ The Transfer verb is used to bridge another party onto the current call.
...
 
-When the called party hangs up, if the `transferCompleteUrl` attribute is specified, the [TransferComplete](../callbacks/transferComplete.md) callback is sent to the `transferCompleteUrl` and
-the BXML returned by that callback is executed on the original call. Verbs following the `<Transfer>` will be ignored when the `transferCompleteUrl` attribute is specified.
+When the called party hangs up, if the `transferCompleteUrl` attribute is specified, the [TransferComplete](../callbacks/transferComplete.md) callback is sent to the `transferCompleteUrl`,
+this callback is also sent if any problem occur with the transfer, such as [rate limiting](../../rateLimits.md).
```
Comment (ghost) [inline voice/bxml/verbs/transfer.md]:
```suggestion
this callback is also sent if any problem occurs with the transfer, such as the callee is busy, doesn't answer, or the call gets [rate limited](../../rateLimits.md).
```

Comment (ghost) [reply voice/bxml/verbs/transfer.md]: I wonder if we should reference this paragraph from the `transferCompleteUrl` field description and say something like "Optional but highly recommended.  See the paragraph below for further details'

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-3838.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-3838.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-3838.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-3838.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-3838.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-3838.s3-website-us-east-1.amazonaws.com/

File: voice/bxml/verbs/transfer.md (hunk 1 above)
Comment (muesono-daitan) [reply voice/bxml/verbs/transfer.md]: Added a link in the field.

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-3838.s3-website-us-east-1.amazonaws.com/

**Review by ghost — APPROVED** [review]
Looks good!

**Pull request merged**